doc/classes/VisualServer.xml

```diff
@@ -2604,6 +2604,15 @@
 			<description>
 			</description>
 		</method>
+		<method name="multimesh_create">
+			<return type="RID">
+			</return>
+			<description>
+			Creates a new multimesh on the VisualServer and returns an [RID] handle.
+			
+			Once finished with your RID, you will want to free the RID using the VisualServer's [code]free_rid()[/code] static method.
```

Comment:
There should be one more level of indentation in the XML (one tab after the level of `<description>`. There should also not be empty lines which will be each treated as a new `<p>` in the HTML.

For `free_rid`, you can refer to the method directly with a hyperlink: "using [method free_rid]".